### ISSUE TITLE
Adding noise

### ISSUE THREAD

**Issue by qo4on** [3b1b/manim#876]
Is it possible to add a small amount of noise with Manim like this thing [Renoiser Modern Master 25%](https://youtu.be/MIgSWhItPoI) does? Why do I ask it? I used it a couple of times and it's great. It brings mush quality improvement to web video. For example, it forces youtube compression algorithms to keep as much details as possible. It forbids youtube to apply hard compression algorithms, which you usually see as small same color squares on the image.

**Comment by kolibril13** [comment]
Yeah, try to make a randomnoise numpy array, then add an ImageMobject from this random image to your scene, scale the image to your scene height and width, and finally set the ImageMobject opacity very low (eg. 0.02)

**Comment by kolibril13** [comment]
I ve not done this before, but I think it should work

**Comment by qo4on** [comment]
This could be a way to go, but renoiser is not just a layer with transparency. The noise there depends on the image in some way.

**Comment by kolibril13** [comment]
Ok, then you have to go with additional after processing software.

**Comment by qo4on** [comment]
The question is how to do this with Manim. I already know how to do this with other software.
